openerp/addons/base/security/base_security.xml

```diff
@@ -44,6 +43,7 @@
         </record>
         <record model="res.groups" id="group_user">
             <field name="implied_ids" eval="[(4, ref('group_no_one'))]"/>
+            <field name="users" eval="[(4, ref('base.user_root'))]"/>
```

Comment:
It was recomputed at each write on res.groups, whatever changed. Thus this second write (implied_ids) fixed the wrongly value of "share" for user_root. Now we only recompute it if "users" is changed.